resources/inspec_gem.rb

```diff
@@ -80,12 +80,12 @@ def compatible_with_client?(gem_version)
     return true if gem_version.nil?
 
     requirement = if chef_15?
-      # Chef-15 requires train 2.0 which was added in Inspec 4
-      Gem::Requirement.new('>= 4')
-    else
-      # min version required to run the audit handler
-      Gem::Requirement.new(['>= 1.25.1'])
-    end
+                    # Chef-15 requires train 2.0 which was added in Inspec 4
```

Comment:
This seems super odd, maybe it's just the diff.

Reply:
I know! Linting wanted `if`, `else` and `end` aligned. That was my quick fix without refactoring the code.